### PR TITLE
Add Jira target docs

### PR DESCRIPTION
**Pull request by odacremolbap**
Add Jira target docs.

- Examples based on the [target's sample](https://github.com/triggermesh/knative-targets/blob/master/samples/jira/500-usage.md).
- Schemas added as links at the doc.

### CODE REVIEW

File: docs/targets/jira.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with was successfully created.
+
+![Bridge status](../images/bridge-status-green.png)
+
+## Event Types
+
+The Jira target accepts these event types:
+
+- `com.jira.issue/create`
+
```
Comment (antoineco) [inline docs/targets/jira.md:40]: Those slashes are quite inconsistent with everything we've done so far. No saying that's not a good pattern, but I'd really love to see us agreeing on _one_ consistent pattern, on only then apply it across the board. Too late to revert and discuss, I imagine?

Comment (odacremolbap) [reply docs/targets/jira.md:40]: The target [is merged](https://github.com/triggermesh/knative-targets/pull/361).
We can open a ticket for accepted event types naming at targets or misc.

**Review by antoineco — APPROVED** [review]

**Review by cab105 — COMMENTED** [review]
Will hold off reviewing until after https://github.com/triggermesh/knative-targets/pull/368 is addressed. However did notice some of the schema definitions need a newline at the end of the file.

**Comment by odacremolbap** [comment]
> Will hold off reviewing until after [triggermesh/knative-targets#368](https://github.com/triggermesh/knative-targets/pull/368) is addressed. However did notice some of the schema definitions need a newline at the end of the file.

@cab105 updated

File: docs/targets/jira.md (hunk 1 above)
Comment (cab105) [reply docs/targets/jira.md:40]: Are we still going with `com.jira.issue.create`? I thought you had merged the target with `io.triggermesh.jira.issue.create`?

File: docs/targets/jira.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+The Jira target accepts these event types:
+
+- `com.jira.issue.create`
+
+The Jira target will create an issue when receiving this event type. The CloudEvent data must contain a Jira issue JSON formated as defined at [this schema](../schemas/jira.issue.json)
+
+Reply contains a a partially filled Jira issue with updated data.
+
+- `com.jira.issue.get`
```
Comment (cab105) [inline docs/targets/jira.md:45]: Same here for `io.triggermesh.jira.issue.get`

File: docs/targets/jira.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+The Jira target will retrieve an issue when receiving this event type. The CloudEvent data must contain a Jira Issue get request  JSON formated as defined at [this schema](../schemas/jira.issue.get.json)
+
+Reply data contains a Jira issue.
+
+- `com.jira.custom`
```
Comment (cab105) [inline docs/targets/jira.md:51]: And here for `io.triggermesh.jira.custom`

**Review by cab105 — APPROVED** [review]

**Pull request merged**